structuredheadings/plugin.js

```diff
@@ -490,7 +490,7 @@
             editor.execCommand("applyHeadingPreset", value);
           }
 
-          if (value !== "restart" && value !== "clear") {
+          if (value !== "restart" && value !== "clear") {structuredheadings/plugin.js
```

Comment:
My money says this change is going to cause problems :)

Time for me to turn off notifications for this repo.

Reply:
I was showing how to contribute to CKEditor. Overlooked it 🤦 .

Thanks @jlward 🙇